src/Ombi/ClientApp/app/requests/tvrequests.component.ts

```diff
@@ -227,13 +229,11 @@ export class TvRequestsComponent implements OnInit, OnDestroy {
         }
     }
     private removeChildRequestFromUi(key: IChildRequests) {
-        this.tvRequests.forEach((val) => {
```

Comment:
Do we no longer need this?

Reply:
Checked in new update removing unused functions and updated type interface on a missed typed